CHANGELOG.next.toml

```diff
@@ -12,35 +12,7 @@
 # author = "rcoh"
 
 [[aws-sdk-rust]]
-message = """
-The `meta`, `environment`, and `dns` Cargo feature flags were removed from `aws-config`.
-The code behind the `dns` flag is now enabled when `rt-tokio` is enabled. The code behind
-the `meta` and `environment` flags is always enabled now.
-"""
-references = ["smithy-rs#961"]
-meta = { "breaking" = true, "tada" = false, "bug" = false }
-author = "jdisanti"
-
-[[aws-sdk-rust]]
-message = "Debug implementation of Credentials will print `expiry` in a human readable way."
-meta = { "breaking" = false, "tada" = false, "bug" = false }
-references = ["smithy-rs#973"]
-author = "rcoh"
-
-[[aws-sdk-rust]]
-message = "Add Route53 customization to trim `/hostedzone/` prefix prior to serialization. This fixes a bug where round-tripping a hosted zone id resulted in an error."
-meta = { "breaking" = false, "tada" = false, "bug" = true }
-references = ["smithy-rs#999", "smithy-rs#143", "aws-sdk-rust#344" ]
-author = "rcoh"
-
-[[aws-sdk-rust]]
-message = "Fix bug where ECS credential provider could not perform retries."
-meta = { "breaking" = false, "tada" = false, "bug" = true }
-references = ["smithy-rs#998", "aws-sdk-rust#359"]
-author = "rcoh"
-
-[[aws-sdk-rust]]
-message = "aws_types::Config is now `Clone`"
+message = "Example for Config builder region function added"
+references = ["smithy-rs#670"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
-references = ["smithy-rs#1002", "aws-sdk-rust#352"]
-author = "rcoh"
```

Comment:
if you look at this diff, you can see that your changelog update deleted other changelog entries

Reply:
Ah sorry, I messed up my branch before I thought these  came from the mess-up. I will try put them back